driver-core/src/main/java/com/datastax/driver/core/ExecutionInfo.java

```diff
@@ -112,8 +113,24 @@ public Host getQueriedHost() {
     }
 
     /**
+     * The speculative execution that completed this query.
+     * <p>
+     * 1 represents the initial, regular execution of the query, 2 represents the first speculative
+     * execution, etc.
+     * <p>
+     * Note that this is different from the number of <em>started</em> executions. For example, if
+     * one speculative execution was triggered, but the initial execution eventually completed
+     * first, this will be 1.
```

Comment:
Wouldn't users wish they could have a more complete vision? I.e know exactly how many spec execs were triggered, and which one completed and which ones were canceled)?